<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LevelMaker</class>
 <widget class="QWidget" name="LevelMaker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LevelMaker</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>Available levels</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>40</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>Selected levels</string>
     </property>
    </widget>
    <widget class="QPushButton" name="MainMenuButton">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>510</y>
       <width>201</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>MainMenu (save changes)</string>
     </property>
    </widget>
    <widget class="QListWidget" name="availableLevels">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>261</width>
       <height>291</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="selectedLevels">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>90</y>
       <width>261</width>
       <height>291</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="SelectButton">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>390</y>
       <width>112</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Select</string>
     </property>
    </widget>
    <widget class="QPushButton" name="DeselectButton">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>390</y>
       <width>112</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Deselect</string>
     </property>
    </widget>
    <widget class="QPushButton" name="EditButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>390</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Edit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="UpButton">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>390</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Up</string>
     </property>
    </widget>
    <widget class="QPushButton" name="DownButton">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>420</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Down</string>
     </property>
    </widget>
    <widget class="QPushButton" name="DeleteButton">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>420</y>
       <width>112</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
    <widget class="QPushButton" name="RenameButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>420</y>
       <width>91</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Rename</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
